server/conf/application.conf

```diff
@@ -180,7 +180,7 @@ play.i18n {
   # adding translations for programs and applications.
   #
   # Alphabetize all languages besides English.
-  langs = [ "en-US", "am", "zh-TW", "ja", "ko", "lo", "om", "ru", "so", "es-US", "tl", "ti" , "vi" ]
+  langs = [ "en-US", "am", "zh-TW", "fr", "ja", "ko", "lo", "om", "ru", "so", "es-US", "tl", "ti" , "vi" ]
```

Comment:
(question) Do we know if all CEs are setting the languages in their deploy config files? If they are not they will suddenly have French come online.

Reply:
I've checked all the production instances and confirmed they configure specific languages I think it's probably helpful on staging / by default to show all the languages that have CiviForm static text translation, but I can send a message confirming? I had just seen russian and some other new languages were added, so adding french would make it so all languages we have support for are included in this list.

Reply:
It's likely I asked the same thing last time, lol. If prod configs are good that's plenty. If there are new languages on staging sites I wouldn't bother email people about it.

Reply:
Sounds good! Thanks!